decidim-core/app/controllers/decidim/notifications_settings_controller.rb

```diff
@@ -10,6 +10,27 @@ def show
       @notifications_settings = form(NotificationsSettingsForm).from_model(current_user)
     end
 
+    def unsubscribe
+      authorize! :update, current_user
```

Comment:
Hi! Looks like this whole block is untested:

![image](https://user-images.githubusercontent.com/111746/34043256-818f394a-e1a0-11e7-8d0a-e9f92959884c.png)

I use the `codecov` extension for Chrome in order to see that in a glance ;)

Reply:
And here's @josepjaume's little secret explaining his will to keep using codecov! :stuck_out_tongue:

Reply:
LOL 🤣

Reply:
You can't say it isn't helpful 🤣

Reply:
I really can't. I inmediately installed it after I saw your screenshot! :laughing: